setup.cfg

```diff
@@ -181,66 +178,26 @@ filterwarnings =
     # is being ignored
     #
     # https://github.com/pytest-dev/pytest-cov/issues/557
+    # It was "fixed" and released but it doesn't seem to be fixed
     ignore:The --rsyncdir command line argument and rsyncdirs config variable are deprecated.:DeprecationWarning
-    # Ignore this on our oldestdeps run that doesn't support the xdist_group marker
```

Comment:
I removed the ones our CI does not seem to raise anymore and added some more info where possible.